amoro-ams/amoro-ams-server/src/main/java/org/apache/amoro/server/dashboard/utils/DesensitizationUtil.java

```diff
@@ -22,11 +22,26 @@
 public class DesensitizationUtil {
   public static String COMMON_SECRET_FORMAT = "********";
 
+  private static final String[] SENSITIVE_CONF_KEYWORDS = {"secret", "token", "password"};
```

Comment:
I am not sure if we should desensitize all properties containing the keywords above.
Or we may better define specific properties we should desensitize on?

Reply:
Good idea, i will try to do that.